### PR TITLE
Set maximum chunk size to 64GB

### PR DESCRIPTION
**Pull request by willyborankin**
Set maximum chunk size to 64GB

### CODE REVIEW

File: repository-commons/src/main/java/io/aiven/elasticsearch/repositories/CommonSettings.java [hunk 1]
```diff
@@ -43,12 +43,17 @@ static String withPrefix(final String key) {
                         Setting.Property.Dynamic
                 );
 
+        // For default settings of AES/CTR which we use the maximum size of the file
+        // with the same IV is 64GB (2^32 * 128 bit per block)
+        // to use 5TB maximum storage limit we need to improve our logic and use more than one IV
+        // In case of AES/GCM maximum size is teh same since it is extension of CTR with hashing,
+        // but from security point of view GCM better
```
Comment (ivanyu) [inline repository-commons/src/main/java/io/aiven/elasticsearch/repositories/CommonSettings.java:50]: We should evaluate the possibility to trade the nonce size for the counter size. E.g. one more byte given to the counter will be enough for 16 TB.

Comment (ivanyu) [reply repository-commons/src/main/java/io/aiven/elasticsearch/repositories/CommonSettings.java:50]: Or just split it into independently encrypted files to not outsmart ourselves 😃

Comment (willyborankin) [reply repository-commons/src/main/java/io/aiven/elasticsearch/repositories/CommonSettings.java:50]: chunk is a separate file the minimum value is 100 MB with unique IV. but to exclude mistakes we need to add upper limit which is  64 GB so far ... if we need bigger files we need to use custom IV generation which must be smaller than the block size 128 bytes

Comment (ivanyu) [reply repository-commons/src/main/java/io/aiven/elasticsearch/repositories/CommonSettings.java:50]: Ok, I think I got how it works.

**Review by ivanyu — APPROVED** [review]

**Pull request merged**